docs/en/administration/data_migration_tool.md

```diff
@@ -17,6 +17,36 @@ The StarRocks Cross-cluster Data Migration Tool is provided by StarRocks Communi
 
 The following preparations must be performed on the target cluster for data migration.
 
+### Enable Legacy Compatibility for Replication
+
+StarRocks may behave differently between the old and new versions, causing problems with cross-cluster data migration. Therefore, you must enable Legacy Compatibility for the target cluster before data migration and disable it after data migration is completed.
+
+1. You can check whether Legacy Compatibility for Replication is enabled by using the following statement:
+
+   ```SQL
+   ADMIN SHOW FRONTEND CONFIG LIKE 'enable_legacy_compatibility_for_replication';
+   ```
+
+   If `true` is returned, it indicates that Legacy Compatibility for Replication is enabled.
+
+2. Dynamically enable Legacy Compatibility for Replication:
+
+   ```SQL
+   ADMIN SET FRONTEND CONFIG("enable_legacy_compatibility_for_replication"="true");
+   ```
+
+3. To prevent Legacy Compatibility for Replication from automatically disabling during the data migration process in case of cluster restart, you also need to add the following configuration item in the FE configuration file **fe.conf**:
+
+   ```Properties
+   enable_legacy_compatibility_for_replication = true
```

Comment:
have these two para been added to FE config doc?

Reply:
done